### ISSUE TITLE
[Quest Bug] Retribution aura bugs tournament jousting dailies

### ISSUE THREAD

**Issue by arestavo** [dalaranwow/dalaran-wow#5745]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
If a paladin has retribution aura on, and it deals damage to the jousting opponent for the tournament dailies (such as The Grand Melee), credit for defeating the jousting opponent will not be given.

This may have a direct bearing on other players being able to "kill steal" off of you while doing these dailies.
**Expected behaviour**: Tell us what should happen instead
Defeating a jousting opponent for the jousting daily should award credit for the win regardless as to whether or not retribution aura is on.
**Steps to reproduce the problem**:

1. Turn on retribution aura as a paladin.
2. Try to complete the jousting daily quests.
3. No credit is given for a jousting win.

**Include proofs for this behaviour**
This is 100% repeatable and easy to verify in game with a paladin.

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by talkaboom** [comment]
I can also add that Devotion Aura reduces the damage from all jousting attacks. It is not supposed to.
Another problem I frequently encounter is that it fails to do any damage when you charge,
